src/main/java/com/feelmycode/parabole/domain/Event.java

```diff
@@ -0,0 +1,136 @@
+package com.feelmycode.parabole.domain;
+
+import static javax.persistence.FetchType.LAZY;
+
+import com.fasterxml.jackson.annotation.JsonBackReference;
+import com.fasterxml.jackson.annotation.JsonManagedReference;
+import java.io.Serializable;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Embedded;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import javax.persistence.UniqueConstraint;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Table(name = "events", uniqueConstraints = {
+    @UniqueConstraint(name = "event_seller_unique", columnNames = {"event_id", "seller_id"})})
+@Getter
+@NoArgsConstructor
+public class Event extends BaseEntity implements Serializable {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "event_id")
+    private Long id;
+
+    @ManyToOne
+    @JoinColumn(name = "seller_id")
+    @JsonBackReference
+    private Seller seller;
+
+    @Column(name = "created_by")
+    private String createdBy; // [ADMIN, SELLER]
+
+    @Column(name = "event_type")
+    private String type; // [FCFS, REAFFLE]
+
+    @Column(name = "event_title")
+    private String title;
+
+    @Column(name = "event_start_at")
+    private LocalDateTime startAt;
+
+    @Column(name = "event_end_at")
+    private LocalDateTime endAt;
```

Comment:
BaseEntity에 있는 부분이라서 extends BaseEntity하고 이 부분 제거해도 되지 않나요?
```suggestion
    @Column(name = "event_type")
    private String type; // [FCFS, REAFFLE]

    @Column(name = "event_title")
    private String title;
```

Reply:
이벤트 등록/수정/삭제 필드는 BaseEntity상속 받았고 해당 부분은 이벤트 실제 '시작일시' 및 '종료일시'라 다른 컬럼이라서 남겨두겠습니다~